apple/MarkdownTextInputDecoratorComponentView.mm

```diff
@@ -48,46 +46,15 @@ - (instancetype)initWithFrame:(CGRect)frame
   return self;
 }
 
-- (void)updateState:(const facebook::react::State::Shared &)state oldState:(const facebook::react::State::Shared &)oldState
+- (void)didAddSubview:(UIView *)subview
 {
-    auto data = std::static_pointer_cast<MarkdownTextInputDecoratorShadowNode::ConcreteState const>(state)->getData();
-
-    if (_decoratorFamily != nullptr) {
-        MarkdownShadowFamilyRegistry::unregisterFamilyForUpdates(_decoratorFamily);
-    }
-
-    _decoratorFamily = data.decoratorFamily;
-    MarkdownShadowFamilyRegistry::registerFamilyForUpdates(_decoratorFamily);
-}
-
-- (void)willMoveToSuperview:(UIView *)newSuperview {
-    if (newSuperview == nil) {
-        MarkdownShadowFamilyRegistry::unregisterFamilyForUpdates(_decoratorFamily);
-      _decoratorFamily = nullptr;
-    }
-
-    [super willMoveToSuperview:newSuperview];
-}
-
-- (void)didMoveToWindow {
-  if (self.superview == nil) {
-    return;
-  }
-
-  NSArray *viewsArray = self.superview.subviews;
-  NSUInteger currentIndex = [viewsArray indexOfObject:self];
-
-  react_native_assert(currentIndex != 0 && currentIndex != NSNotFound && "Error while finding current component.");
-  UIView *view = [viewsArray objectAtIndex:currentIndex - 1];
-
-  react_native_assert([view isKindOfClass:[RCTTextInputComponentView class]] && "Previous sibling component is not an instance of RCTTextInputComponentView.");
-  _textInput = (RCTTextInputComponentView *)view;
+  react_native_assert([subview isKindOfClass:[RCTTextInputComponentView class]] && "Previous sibling component is not an instance of RCTTextInputComponentView.");
+  _textInput = (RCTTextInputComponentView *)subview;
   _backedTextInputView = [_textInput valueForKey:@"_backedTextInputView"];
 
   _markdownUtils = [[RCTMarkdownUtils alloc] init];
-  react_native_assert(_markdownStyle != nil);
```

Comment:
Why do we need to remove this assert?

Reply:
Same reason the other comment mentioned - it may be uninitialized here due to when new props are set.